benchmark/tpch.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB TPC-H 50G Performance Test Report V2.0
+category: benchmark
+draft: true
+---
+
+# TiDB TPC-H 50G Performance Test Report
+
+## Test purpose
+
+The test aims at comparing the performance between TiDB v1.0 and v2.0 in OLAP scenario.
```

Comment:
-> This test aims to compare the performances of TiDB 1.0 and TiDB 2.0 in the OLAP scenario.